### PR TITLE
Example of refreshing table contents

### PR DESCRIPTION
**Pull request by mohsenomidi**
Simple usage real-time refreshing table

### CODE REVIEW

File: samples/refresh_table.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    if (std::cin.get() == 10) {
+      keep_running = false;
+    }
+  }
+  return NULL;
+}
+
+int main() {
+  pthread_t tId;
+  (void)pthread_create(&tId, 0, userInput_thread, 0);
```
Comment (p-ranav) [inline samples/refresh_table.cpp:21]: Please change to `std::thread`. `pthread` will not compile in Windows

File: samples/refresh_table.cpp [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#include <chrono>
+#include <random>
+#include <tabulate/table.hpp>
+#include <thread>
+
+using namespace tabulate;
+using Row_t = std::vector<variant<std::string, const char *, Table>>;
+static volatile bool keep_running = true;
```
Comment (p-ranav) [inline samples/refresh_table.cpp:8]: Change to `std::atomic_bool`. `keep_running` is modified in a separate `userInput_thread` without locking.

**Review by p-ranav — CHANGES_REQUESTED** [review]

File: samples/refresh_table.cpp (hunk 1 above)
Comment (mohsenomidi) [reply samples/refresh_table.cpp]: Done.

**Comment by p-ranav** [comment]
Travis build is failing, please check the log here: https://travis-ci.com/github/p-ranav/tabulate/jobs/394791629

You're missing some header includes for thread, chrono, random, vector etc.

File: samples/refresh_table.cpp (hunk 2 above)
Comment (mohsenomidi) [reply samples/refresh_table.cpp]: Done

**Comment by mohsenomidi** [comment]
It exists but in my system it compiled without those headers, I think it already included in the original single header file, any way, i will reverted this change and put back the headers.

**Comment by mohsenomidi** [comment]
All checks done.
@p-ranav Please review.

**Comment by p-ranav** [comment]
Thanks!

**Pull request merged**